example/counter/pubspec.yaml

```diff
@@ -10,7 +13,7 @@ dev_dependencies:
     path: ../../
   flutter_test:
     sdk: flutter
-  pedantic: ^1.8.0+1
+  pedantic: 1.10.0
```

Comment:
I understand that we want null safety in the dependencies, but rather than this specific version shouldn't it be:
`pedantic: ^1.10.0`

Reply:
Versions of dependencies in the demo project aren't  critical, but I agree with other your comments